terraform/environment/terraform.tfvars.json

```diff
@@ -9,7 +9,8 @@
       ],
       "app": {
         "env": {
-          "app_public_url": "https://opg-lpa-fd-prototype.apps.live.cloud-platform.service.justice.gov.uk"
+          "app_public_url": "",
```

Comment:
This needs a rethink but didn't want to bloat this PR any further

Reply:
could you revert all of the changes you've done to the terraform because auth no longer works